pyserini/search/_dsearcher.py

```diff
@@ -0,0 +1,198 @@
+from typing import List, Optional
```

Comment:
I think PEP8 wants a different ordering of imports?

Reply:
We probably need to add these to the pyserini requirements and lib dependencies...

Reply:
sure